conf/zeppelin-site.xml.template

```diff
@@ -410,6 +410,12 @@
   <description>Anonymous user allowed by default</description>
 </property>
 
+<property>
+  <name>zeppelin.username.force.lowercase</name>
+  <value>false</value>
+  <description>Force convert username case to lower case, useful for Active Directory. Default is not to change case</description>
```

Comment:
could you elaborate more on `, useful for Active Directory`? say `as Active Directory/LDAP are case-insensitive`?
also, isn't this applicable to LDAP too?

Reply:
yes, this is very much applicable to LDAP. I'll update in the next commit.